examples/threathunter/threat_intelligence/feed_helper.py

```diff
@@ -5,14 +5,18 @@
 
 from datetime import datetime, timedelta, timezone
 
+log = logging.getLogger(__name__)
+
 
 class FeedHelper():
     def __init__(self, start_date, size_of_request_in_minutes):
         self.size_of_request_in_minutes = size_of_request_in_minutes
         if isinstance(start_date, datetime):
             self.start_date = start_date.replace(tzinfo=timezone.utc)
-        else:
+        elif isinstance(start_date, string):
             self.start_date = datetime.strptime(start_date, "%Y-%m-%d %H:%M:%S").replace(tzinfo=timezone.utc)
+        else:
+            log.error(f"Start_date must be a string or datetime object. Received a start_time config value with unsupported type: {type(start_date)}")
```

Comment:
You will need to break or raise an Error otherwise the code will fail after

Reply:
Still working on the dev, I haven't tested locally again yet. There's probably a couple things that'll need to change. Thank you for the callout!